functions/fn_initServer.sqf

```diff
@@ -34,5 +34,11 @@ params ["_logicModule"];
 // init finished, broadcast
 _logicModule setVariable ["SCAR_UCM_initialized", true, true];
 
+//create worker groups for every sife
+UCM_west = createGroup west;
+UCM_east = createGroup east;
+UCM_ind  = createGroup resistance;
+UCM_civ  = createGroup civilian;
```

Comment:
This is wrong and terrible I'm sure. If this were done, surely a group should be created by the logic module for the workers to join.